[2024-09-07T23:01:07.973+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-07T23:01:09.004+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: welcome_dag.print_date scheduled__2024-09-07T23:00:01+00:00 [queued]>
[2024-09-07T23:01:09.203+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: welcome_dag.print_date scheduled__2024-09-07T23:00:01+00:00 [queued]>
[2024-09-07T23:01:09.210+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-07T23:01:09.673+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): print_date> on 2024-09-07 23:00:01+00:00
[2024-09-07T23:01:09.772+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=601) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-07T23:01:09.760+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'welcome_dag', 'print_date', 'scheduled__2024-09-07T23:00:01+00:00', '--job-id', '447', '--raw', '--subdir', 'DAGS_FOLDER/Awelcome_dags.py', '--cfg-path', '/tmp/tmpz6cdz3jr']
[2024-09-07T23:01:09.781+0000] {standard_task_runner.py:72} INFO - Started process 623 to run task
[2024-09-07T23:01:09.784+0000] {standard_task_runner.py:105} INFO - Job 447: Subtask print_date
[2024-09-07T23:01:14.389+0000] {task_command.py:467} INFO - Running <TaskInstance: welcome_dag.print_date scheduled__2024-09-07T23:00:01+00:00 [running]> on host 13d1dadf4f15
[2024-09-07T23:01:15.944+0000] {job.py:229} INFO - Heartbeat recovered after 13.34 seconds
[2024-09-07T23:01:17.597+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='welcome_dag' AIRFLOW_CTX_TASK_ID='print_date' AIRFLOW_CTX_EXECUTION_DATE='2024-09-07T23:00:01+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-07T23:00:01+00:00'
[2024-09-07T23:01:17.600+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-07T23:01:17.654+0000] {logging_mixin.py:190} INFO - Today is 2024-09-07
[2024-09-07T23:01:17.656+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-07T23:01:18.423+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-07T23:01:18.425+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=welcome_dag, task_id=print_date, run_id=scheduled__2024-09-07T23:00:01+00:00, execution_date=20240907T230001, start_date=20240907T230109, end_date=20240907T230118
[2024-09-07T23:01:19.295+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-07T23:01:19.807+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2024-09-07T23:01:21.448+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-07T23:01:21.477+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
